Cargo.toml

```diff
@@ -12,6 +12,7 @@ edition = "2021"
 publish = false
 
 [workspace.dependencies]
-libcnb = "0.11"
-libcnb-test = "0.11"
-libherokubuildpack = "0.11"
+libcnb = "0.11.5"
+libcnb-test = "0.11.5"
+libherokubuildpack = "0.11.5"
+toml = "0.7.1"
```

Comment:
I've intentionally switched several repos to only specifying the major versions in `Cargo.toml`, given that (a) even the "full" versions equate to a version range, (b) it avoids some of the conflicts between Dependabot PRs, meaning less rebasing needed.

I know in this case if we're being strict, then to get TOML 7+ you do need libcnb 0.11.5+, however I guess that perhaps indicates the libcnb release maybe should have been 0.12.0 instead?

Reply:
I think for libraries (eg libcnb) then adding a minimum bound for the version (as the full strings do) makes sense since libraries don't make use of the upstream `Cargo.lock`, however for end applications (like our buildpacks), it seems fine to rely on `Cargo.lock` for the patch version?

Reply:
> perhaps indicates the libcnb release maybe should have been 0.12.0 instead?

Following that logic, wouldn't all dependency bumps in libcnb cause non-patch version bumps of libcnb then? Seems weird to me.

Honestly, I think I'm using the (seemingly) full versions because that's what I "grew up with" in the JVM ecosystem where it is common practice and lockfiles don't exist. Not that this is a _good_ reason, but _a_ reason. :) 

In any case, I just realised that this PR actually changes the versions to use the full strings so I'm happy to revert back to the version strings without the path versions specified. If we want to, we can discuss this separately outside of this PR.

Reply:
resolved via 4d10959.

Reply:
> Following that logic, wouldn't all dependency bumps in libcnb cause non-patch version bumps of libcnb then?

No, only ones that can cause breaking changes. My understanding was that the re-exported `toml` from libcnb meant some end users of libcnb had to keep their `toml` version in sync otherwise errors ensued.